ecs-agent/acs/session/attach_resource_responder.go

```diff
@@ -194,7 +205,7 @@ func validateAttachmentAndReturnProperties(message *ecsacs.ConfirmAttachmentMess
 		return attachmentProperties, nil
 	}
 
-	// For "EphemeralStorage" and "ElasticBlockStorage", ACS is using resourceType to indicate its attachment type.
+	// For legacy EBS volumes("EphemeralStorage" and "ElasticBlockStorage"), ACS is using resourceType to indicate its attachment type.
```

Comment:
nit: For my understanding, is it appropriate referring to these storage means as legacy? Are we planning to deprecate them at any point already?

Reply:
> Are we planning to deprecate them at any point already?

I don't think so. I think we will continue to support both but won't change them much.

I will confirm with @fierlion  since this comment is from this PR https://github.com/aws/amazon-ecs-agent/pull/3911/files/9597f968e447852d88a3fb7d75a7a550fba24ffb#r1327869451

Reply:
non-blocking: My comment's intent was to remove the words EphemeralStorage altogether.  This is a nit but we should follow up and fix this comment.